sprint4/.env.example

```
@@ -0,0 +1,2 @@
+DATABASE_URL=postgresql://postgres:postgres@localhost:5432/panda-market
```

env example에는 명확한 값을 안 넣어주는 것이 좋습니다
```suggestion
DATABASE_URL=postgresql://postgres:postgres@localhost:5432/database
```